ImperatorToCk3Tests/ImperatorWorldTests/Provinces/ProvincesTests.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gtest/gtest.h"
+#include <sstream>
+
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Provinces.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Province.h"
+
+TEST(ImperatorWorld_ProvincesTests, provincesDefaultToEmpty)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Provinces provinces(input);
+
+	ASSERT_TRUE(provinces.getProvinces().empty());
+}
+
+TEST(ImperatorWorld_ProvincesTests, provincesCanBeLoaded)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "42={}\n";
+	input << "43={}\n";
+	input << "}";
+
+	const ImperatorWorld::Provinces provinces(input);
+	const auto& provinceItr = provinces.getProvinces().find(42);
+	const auto& provinceItr2 = provinces.getProvinces().find(43);
+
+	ASSERT_EQ(42, provinceItr->first);
+	ASSERT_EQ(42, provinceItr->second->getID());
+	ASSERT_EQ(43, provinceItr2->first);
+	ASSERT_EQ(43, provinceItr2->second->getID());
+}
```

Comment:
Missing tests for linkage. 
Show what happens when linkage breaks. CK2 has some relevant test templates for this, as well as for checking the error messages that are output.

Reply:
Done, can you check them? Especially the dots vs arrows.

Reply:
looks solid. All green here, might want to wait for @Idhrendur in case I missed something.